### ISSUE TITLE
The shared login-free window lacks the voice input function

### ISSUE THREAD

**Issue by WyHy** [labring/FastGPT#558]
我是基于docker-compose使用的4.6.3版本，我遇见两个问题：
1. 我发现在应用配置页面右侧的对话窗口中有语音输入的按钮，但是点击好像没啥反应，这个是还没启用吗？
2. 分享出去的免登录窗口中我看就没有这个语音输入按钮了，就一个文本输入功能，分享页面是没有语音输入功能吗？

**Comment by c121914yu** [comment]
> Bot detected the issue body's language is not English, translate it automatically. 👯👭🏻🧑‍🤝‍🧑👫🧑🏿‍🤝‍🧑🏻👩🏾‍🤝‍👨🏿👬🏿         
----    
I am using version 4.6.3 based on docker-compose, and I encountered two problems:
1. I found that there is a voice input button in the dialog window on the right side of the application configuration page, but there seems to be no response when I click it. Is this not enabled yet?
2. I see that there is no voice input button in the login-free window for sharing. There is only a text input function. Does the sharing page not have a voice input function?
